### PR TITLE
xt new endpoints

### PR DESCRIPTION
**Pull request by ndubel**
https://doc.xt.com/#user_center_documentationrestApi

### CODE REVIEW

**Comment by carlosmiei** [comment]
@ndubel thanks, will merge it soon

File: ts/src/xt.ts [hunk 1]
```diff
@@ -213,6 +214,8 @@ export default class xt extends Exchange {
                         'post': {
                             'order': 0.2,
                             'withdraw': 1,
+                            'balance/transfer': 1,
+                            'balance/account/transfer': 1,
```
Comment (carlosmiei) [inline ts/src/xt.ts:218]: I will check it, because I think we can probably implement `transfer` with this one

Comment (ndubel) [reply ts/src/xt.ts:218]: I've already checked that it works, yes, good for transfer

Comment (carlosmiei) [reply ts/src/xt.ts:218]: > I've already checked that it works, yes, good for `transfer`

@ndubel added 🙂

**Comment by carlosmiei** [comment]
DEMO
```
p xt transfer "USDT" 1 spot swap          
Python v3.10.9
CCXT v3.0.100
xt.transfer(USDT,1,spot,swap)
{'amount': None,
 'currency': None,
 'datetime': None,
 'fromAccount': None,
 'id': '226972662307830528',
 'info': {'ma': [], 'mc': 'SUCCESS', 'rc': '0', 'result': '226972662307830528'},
 'status': None,
 'timestamp': None,
 'toAccount': None}
```
```
 p xt transfer "USDT" 1 swap spot
Python v3.10.9
CCXT v3.0.100
xt.transfer(USDT,1,swap,spot)
{'amount': None,
 'currency': None,
 'datetime': None,
 'fromAccount': None,
 'id': '226972715232313088',
 'info': {'ma': [], 'mc': 'SUCCESS', 'rc': '0', 'result': '226972715232313088'},
 'status': None,
 'timestamp': None,
 'toAccount': None}
```

**Review by carlosmiei — APPROVED** [review]

**Comment by ndubel** [comment]
maybe you also want to hardcode new errors?
https://doc.xt.com/#documentationerrorCode 😀

**Pull request merged**